src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/probation/ProbationEventListenerIntTest.kt

```diff
@@ -142,6 +145,41 @@ class ProbationEventListenerIntTest : MessagingMultiNodeTestBase() {
     checkTelemetry(CPR_RECORD_CREATED, mapOf("SOURCE_SYSTEM" to "DELIUS", "CRN" to crn))
   }
 
+  @Test
+  fun `should log when message processing fails`() {
+    val crn = randomCRN()
+    stub500Response(crn, STARTED, "failure")
+    stub500Response(crn, STARTED, "failure")
+    stub500Response(crn, STARTED, "failure")
+    val crnType = PersonIdentifier("CRN", crn)
+    val personReference = PersonReference(listOf(crnType))
+
+    val domainEvent = DomainEvent(
+      eventType = NEW_OFFENDER_CREATED,
+      detailUrl = createDeliusDetailUrl(crn),
+      personReference = personReference,
+      additionalInformation = null,
+    )
+    val messageId = publishDomainEvent(NEW_OFFENDER_CREATED, domainEvent)
+
+    probationEventsQueue!!.sqsClient.purgeQueue(
+      PurgeQueueRequest.builder().queueUrl(probationEventsQueue!!.queueUrl).build(),
+    ).get()
+    probationEventsQueue!!.sqsDlqClient!!.purgeQueue(
+      PurgeQueueRequest.builder().queueUrl(probationEventsQueue!!.dlqUrl).build(),
+    ).get()
+
```

Comment:
A bit messy but we need to clear this message off the queue otherwise it will be reprocessed during the next test!